Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM php:5.6
+
+RUN echo "date.timezone = UTC" > /usr/local/etc/php/conf.d/date.ini
+
+VOLUME /app
+WORKDIR /app
+
+ENTRYPOINT ["./app/console"]
+CMD ["server:run", "0.0.0.0:8000"]
```

Comment:
`server:run` is designed to run on the developer's machine which probably have an other webserver listening on port 80.
This issue does not exists in a containers.